user/nvidia/template.py

```diff
@@ -0,0 +1,190 @@
+pkgname = "nvidia"
+pkgver = "560.35.03"
+pkgrel = 0
+archs = ["x86_64"]
+hostmakedepends = ["gcompat", "patchelf"]
+depends = [
+    # linked to
+    "libgcc-chimera",
+    "gcompat",
+    "openssl",
+
+    # executed when used
+    "nvidia-modprobe",
+
+    # used in egl icd
+    "nvidia-settings",
```

Comment:
im kind of unsure if they should remain as is, or be subpackages of nvidia itself, since default application profiles and nvidia-settings's `.desktop` file is from the nvidia package..

Reply:
As `nvidia-settings` is built from source, it feels cleaner to have it as a separate package template :thinking: (at least [Arch Linux does it that way](https://gitlab.archlinux.org/archlinux/packaging/packages/nvidia-settings/-/blob/main/PKGBUILD?ref_type=heads), and looks like they use the icon and .desktop file from the source and not from the `.run` file)

Also by merging the two templates you would have to explicitly describe in which sub-module the files have to be put, and you would not take advantage of cports' auto-split feature (documentation is moved to `*-man`, binaries to `*-bin`, libraries to `*-libs`...)

Reply:
oh, i completely missed the doc directory existing, okay, yeah that's nice, i'll keep them separate